genrl/deep/agents/base.py

```diff
@@ -46,40 +72,56 @@ def __init__(
             set_seeds(self.seed, self.env)
 
     def _create_model(self) -> None:
```

Comment:
Not very sure about that. Issues can arise in case there need to be some additional inputs to be passed to the `_create_model` class. I'd preferably keep this to the end after all other algos are refactored.

Reply:
Ok